src/examples/PdfDownload/PdfDownload.js

```diff
@@ -0,0 +1,23 @@
+import * as FormioLoader from "../../helpers/FormioLoader";
+
+export function PdfDownload() {
+  const div = document.createElement("div");
+  setTimeout(() => {
+    FormioLoader.initFormio();
+  });
+  // emulate Squiz Matrix render the formio container in asset #249261
+  // please note that in the template asset, i've add the ^htmlentities to render pdf-download-message
+  // ie. data-formio-pdf-download-message="%asset_metadata_pdf-download-message^htmlentities%"
+  // so the rich text value is htmlentities encoded and can be passed as data attribute with double quotes
+  div.innerHTML = `
+        <div id="formio" 
```

Comment:
does this example show the use of pdf-download?
as i see data-formio-download-pdf is missing in the div

Reply:
Yes, the example is there in the examples folder and it is working.